src/components/tables/DataTable/pagination/Pagination.tsx

```diff
@@ -0,0 +1,124 @@
+/* Copyright (c) Fortanix, Inc.
+|* This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of
+|* the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import cx from "classnames";
+
+import { Icon } from "../../../graphics/Icon/Icon.tsx";
+import { Input } from "../../../forms/controls/Input/Input.tsx";
+
+import {
+  type PageSizeOption,
+  PaginationSizeSelector,
+} from "./PaginationSizeSelector.tsx";
+import { useTable } from "../DataTableContext.tsx";
+
+import "./Pagination.scss";
+import { Button } from "../../../actions/Button/Button.tsx";
```

Comment:
Please use single quotes for the above. Also if you can move the `Button` import to be grouped with Icon and Input.